maven_lambda/__init__.py

```diff
@@ -24,6 +24,9 @@
     'm': 'http://maven.apache.org/POM/4.0.0',
 }
 
+POM_TIMESTAMP = '%Y%m%d%H%M%S'
+SNAPSHOT_FILE_TIMESTAMP = '%Y%m%d.%H%M%S'
```

Comment:
Nit: Maybe s/`SNAPSHOT_FILE_TIMESTAMP`/`SNAPSHOT_FILENAME_TIMESTAMP` to clarify even more that we're talking about the filenames here?

Reply:
I'm sorry, I missed that comment before merging. Yeah, `FILENAME` works too. I'll put it in a follow up